nodes/alarm_server.py

```diff
@@ -190,7 +196,7 @@ def set_alarm(self, srv):
 
     def get_alarm(self, srv):
         ''' Either returns the alarm request if it exists or a blank alarm '''
-        rospy.logdebug("Got request for alarm: {}".format(srv.alarm_name))
+        rospy.loginfo("Got request for alarm: {}".format(srv.alarm_name))
```

Comment:
I was thinking you normally wouldn't want to see this message, only if you intentionally switched to debug.